### ISSUE TITLE
Scroll Popup Window Bug

### ISSUE THREAD

**Issue by fmevans** [bookingactivities/booking-activities#85]
I've found what I think is a bug in the Booking Activities plugin.

At the 'bookings' page, if you hover over an event, a sub-window pops up showing made bookings. To scroll that list, you move your cursor into that window which pops up.

![scrolling_possible](https://user-images.githubusercontent.com/30388566/94987989-07c1b000-0562-11eb-9c71-4ee655bb5121.jpg)

However, if an event appears _below_ another event on the calendar, then when you try to move your cursor onto its pop-up window, its window disappears and it is not then possible to scroll through its own list. This is because there is a gap between the event being hovered over, and its pop-up window. In that gap is the event above, which then throws up its pop-up window!

![scrolling_not_possible](https://user-images.githubusercontent.com/30388566/94988029-35a6f480-0562-11eb-9680-7e9dff49438b.jpg)

Thus it is not currently possible to scroll through a list of attendees, if those attendees belong to an event positioned below another event on the calendar.

**Comment by yoancutillas** [comment]
Indeed that's quite tricky, I will fix it in the next version, thank you for your report!

Temporary fix:
1. Add this code in [booking-activities/js/booking-system.js line 194](https://github.com/bookingactivities/booking-activities/blob/1.8.9/js/booking-system.js#L194)
```
if( typeof bookacti_remove_mouseover_tooltip_monitor !== 'undefined' ) { 
	if( bookacti_remove_mouseover_tooltip_monitor ) { clearTimeout( bookacti_remove_mouseover_tooltip_monitor ); }
}
```

2. Replace `booking-system.min.js` with `booking-system.js` in booking-activities/booking-activities.php

3. Refresh the bookings page with CTRL+F5 (to clear the browser cache)

**Comment by yoancutillas** [comment]
Released in 1.9.0, currently in [beta](https://booking-activities.fr/en/blog/participate-in-booking-activities-and-add-ons-beta-tests/).
